spec/ralyxa/configuration_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'ralyxa/configuration'
+
+RSpec.describe Ralyxa::Configuration do
+  subject { Ralyxa::Configuration.new }
```

Comment:
Can we use a named `subject`? http://blog.davidchelimsky.net/blog/2012/05/13/spec-smell-explicit-use-of-subject/

Reply:
I'll make this change and update the PR